test/python/qobj/test_qobj.py

```diff
@@ -67,7 +67,7 @@ def setUp(self):
         }
 
         self.bad_qobj = copy.deepcopy(self.valid_qobj)
-        self.bad_qobj.experiments = None  # set experiments to None to cause the qobj to be invalid
+        self.bad_qobj.experiments = []
```

Comment:
When you keep it `None` it changes how ` validate_qobj_against_schema` fails. Instead of failing validation because experiments are required in the json schema `"TypeError: 'NoneType' object is not iterable"` gets raised by https://github.com/Qiskit/qiskit-terra/pull/3383/files#diff-22fb2e08d2f88085b56206a44afb2292R741 because it can't iterate over a NoneType. The only test using this (`test_simjob_raises_error_when_sending_bad_qobj`) specifically was looking for a `SchemaValidationError` so it seemed easier to change this to `[]`.